src/jump_model_utils.jl

```diff
@@ -0,0 +1,147 @@
+########### JuMP model utils #########
+
+mutable struct NumericalBounds
+    min::Float64
+    max::Float64
+    min_index::Any
+    max_index::Any
+end
+
+NumericalBounds() = NumericalBounds(Inf, -Inf, nothing, nothing)
```

Comment:
We should add at the top of the file that is based on the code in SDDP and reference the file and commit.

Reply:
Yup, I'll add that

Reply:
Don't bother. I approve the release of this under the current PowerSimulations.jl license without the copyright notice.